### PR TITLE
[Service] util to init gdbus module

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix log message, and add common function to initialize gdbus module.

### CODE REVIEW

**Review by songgot — APPROVED** [review]
LGTM :+1:

File: daemon/gdbus-util.c [hunk 1]
```diff
@@ -141,3 +141,18 @@ gdbus_put_system_connection (void)
...
   g_clear_object (&g_dbus_sys_conn);
 }
+
+/**
+ * @brief Common function to initailize the DBus module.
+ */
+void
+gdbus_initialize (void)
+{
+  GError *err = NULL;
```
Comment (anyj0527) [inline daemon/gdbus-util.c:151]: How about `g_autoptr (GError) = NULL;` for more short code?

File: daemon/gdbus-util.c [hunk 2]
```diff
@@ -141,3 +141,18 @@ gdbus_put_system_connection (void)
 {
   g_clear_object (&g_dbus_sys_conn);
 }
+
+/**
+ * @brief Common function to initailize the DBus module.
```
Comment (anyj0527) [inline daemon/gdbus-util.c:146]: minor typo `initailize` here and in gdbus-util.h

**Review by anyj0527 — APPROVED** [review]
LGTM with some insignificant comments

File: daemon/gdbus-util.c (hunk 2 above)
Comment (jaeyun-jung) [reply daemon/gdbus-util.c]: updated

File: daemon/gdbus-util.c (hunk 1 above)
Comment (jaeyun-jung) [reply daemon/gdbus-util.c:151]: For future work, I am not sure another code for initializing ml-agent module which requires gerror ptr. Let's keep this in this time.

**Pull request merged**